represetnation_labels/useful_functions.py

```diff
@@ -0,0 +1,93 @@
+import json
+import re
+
+def change_format(og_dict,name):
+    new_dict = {name: list(og_dict.keys()),
+                'values': list(og_dict.values())}
+    return new_dict
+
+def eth_change_format(eth_dict,name):
+    eth_dict_alter = eth_dict.copy()
+    eth_dict_alter['Asian']['Chinese'] = eth_dict_alter['Chinese']['Chinese']
+    del eth_dict_alter['Chinese']
+    new_dict = {name:list(eth_dict_alter.keys()),
+                'values':[sum(v.values()) for v in eth_dict_alter.values()]}
+    return new_dict
+
+def multi_eth_change_format(eth_dict):
+    if list(eth_dict.keys()) == ['White', 'Black', 'Asian', 'Chinese', 'Mixed', 'Other', 'Missing']:
+        return eth_change_format(eth_dict,'Ethnicity')
+    else:
+        dict_copy = {k: eth_change_format(v, k) for k, v in eth_dict.items()}
+        dict_copy = {str(k) + ' values': v['values'] for k, v in dict_copy.items()}
+        dict_copy['Ethnicity'] = ['White', 'Black', 'Asian', 'Mixed', 'Other', 'Missing']
+        return dict_copy
+
+def multi_change_format(multi_dict,name):
+    fst_key = list(multi_dict.keys())[0]
+    variable = list(multi_dict[fst_key].keys())
+    new_dict = {str(k) + ' values':[values for values in v.values()] for k,v in multi_dict.items()}
+    new_dict[name] = variable
+    return new_dict
+
+def get_percents(vals):
+    percents = [round(i/sum(vals[:-1])*100,1) for i in vals[:-1]]
```

Comment:
how come you dont include the last element in vals?

Reply:
because thats the missing element which we dont want to plot